Write remaining two subsubsections of 4.2 (#590)
fpvandoorn/carleson: paper/main.tex

It is therefore tempting to conclude that $J$ must be a strict superset of $I$.
This might not be the case in an arbitrary metric space,
since there is no guarantee that there are points
in an annulus with specified inner and outer radii.

This also means that for two cubes $I$ and $J$, if $I\subseteq J$ as sets,
then we cannot assume that $s(I)\le s(J)$.
The first version of the blueprint incorrectly described the data in a cube as just a subset of $X$,
and the scale function $s$ as a function on these subsets.
This implicitly assumed that $s(I)$ is only determined by $I$, viewed as a subset of $X$.
This was fixed by including the scale as part of the data of the cube.

In the blueprint we kept using the notation ``$I \subset J$'' for the inequality between cubes,
even though that actually included the assumption that $s(I) \le s(J)$. % todo: fix in paper.
This change had no further impact on downstream proofs, except for minor changes to lemma statements.

We thank Georges Gonthier for finding this inaccuracy in the blueprint.

\subsubsection{Just one top cube}

We made a second change to the definition of a grid structure, also suggested by Georges Gonthier.

> The first definition didn't have the axiom that there exists a top cube,

Reviewer: ```suggestion
The first definition did not include the axiom that there exists a top cube,
```
